src/com/ableton/VirtualEnvList.groovy

```diff
@@ -0,0 +1,19 @@
+package com.ableton
+
+
+class VirtualEnvList implements Serializable {
+  @SuppressWarnings('FieldTypeRequired')
+  def script
+  Map<String, VirtualEnv> venvs = [:]
+
+  void add(String python) {
+    String pythonName = python.split('/').last().capitalize()
```

Comment:
In which case is there a `/` and why are we capitalizing the name?

Reply:
@ala-ableton The name shouldn't be capitalized anymore, that made more sense when we had `parallelBranches`. The `/` is necessary when a full path is passed as the `python` argument.

Reply:
Is it really necessary? What is the problem with having `"/usr/bin/python3.7"` as a key in the map? It's only a string.

Reply:
Since the map keys aren't even used for now, maybe it makes more sense to just have this be a regular List.

Reply:
Even better!